hathor/p2p/states/hello.py

```diff
@@ -58,6 +59,21 @@ def _get_hello_data(self) -> Dict[str, Any]:
             'settings_dict': get_settings_hello_dict(),
             'capabilities': protocol.node.capabilities,
         }
+        if self._has_sync_version_capability():
+            data['sync_versions'] = [x.value for x in self._get_sync_versions()]
+        return data
+
+    def _has_sync_version_capability(self) -> bool:
+        return settings.CAPABILITY_SYNC_VERSION in self.protocol.node.capabilities
+
+    def _get_sync_versions(self) -> Set[SyncVersion]:
+        if self._has_sync_version_capability():
+            connections_manager = self.protocol.connections
+            assert connections_manager is not None
+            return connections_manager.get_sync_versions()
+        else:
+            # XXX: this is to make it easy to simulate old behavior if we disable the sync-version capability
+            return {SyncVersion.V1}
```

Comment:
Isn't it better to do this in the `connections_manager.get_sync_versions()`?

Reply:
I agree, as well as `_has_sync_version_capability`.

Reply:
Done.

Reply:
Great! Thanks!